src/App.vue

```diff
@@ -12,7 +12,7 @@ import HelloWorld from './components/HelloWorld.vue'
 
```

Comment:
You can delete everything inside of the header tag in App.vue. It is just sample code that Vue adds when you start a project.

The template can just look something like this.

```
<template>
  <RouterView />
</template>
```